common/cache/domainCache.go

```diff
@@ -901,6 +902,11 @@ var SampleRetentionKey = "sample_retention_days"
 // SampleRateKey is key to specify sample rate
 var SampleRateKey = "sample_retention_rate"
 
+// JitteredWorkflowDeletionHoursKey is key to determine the number of hours
+// generated deletion tasks should be jittered over. Should be a
+// parsable number or else it defaults to 0
+const JitteredWorkflowDeletionHoursKey = "jittered_workflow_deletion_hours"
```

Comment:
Sorry for being late to the party here, but if we need the jitter to be any more granular this might be difficult. I *don't* have a huge amount of context on the feature, but perhaps seconds or minutes might be a better level of granularity?

Reply:
a thought: declaration of jitter in hours is OK, but imho, we should calculate jitter in minute (or even second) resolution to avoid deletion spikes. Does it make sense?

Reply:
I can make it randomly jitter over a minute instead 👍👍

Reply:
on the second thought: if we declare everything in minutes, this will bring more clarity on what to expect from this jitter